scripts/pre-commit.git-lint.sh

```diff
@@ -14,8 +14,13 @@
 # limitations under the License.
 
 # First part return the files being commited, excluding deleted files.
-git diff-index -z --cached HEAD --name-only --diff-filter=ACMRTUXB |
-xargs --null --no-run-if-empty git lint;
+DIFF_FILES=$(git diff-index -z --cached HEAD --name-only --diff-filter=ACMRTUXB)
```

Comment:
Would it be possible to convert this to a python script and store those results as variables? Or does the hook have to be a shell script?